settings.gradle

```diff
@@ -1,5 +1,6 @@
 plugins {
     id "com.gradle.enterprise" version "3.13.1"
+    id "org.gradle.toolchains.foojay-resolver-convention" version "0.4.0"
```

Comment:
latest is `0.5.0`

Reply:
Fixed https://github.com/gradle/gradle-build-scan-quickstart/pull/48